CoreFoundation/Base.subproj/CFRuntime.h

```diff
@@ -195,31 +195,23 @@ typedef struct __CFRuntimeBase {
     uintptr_t _cfisa;
     uintptr_t _swift_rc;
     // This is for CF's use, and must match _NSCFType layout
-#if defined(__LP64__) || defined(__LLP64__)
     _Atomic(uint64_t) _cfinfoa;
-#else
-    _Atomic(uint32_t) _cfinfoa;
-#endif
 } CFRuntimeBase;
 
-#if defined(__LP64__) || defined(__LLP64__)
 #define INIT_CFRUNTIME_BASE(...) {0, _CF_CONSTANT_OBJECT_STRONG_RC, 0x0000000000000080ULL}
-#else
-#define INIT_CFRUNTIME_BASE(...) {0, _CF_CONSTANT_OBJECT_STRONG_RC, 0x00000080UL}
-#endif
 
 #else
 
 typedef struct __CFRuntimeBase {
     uintptr_t _cfisa;
-#if defined(__LP64__) || defined(__LLP64__)
```

Comment:
Because the PR this one reverts did as well. 

The PR that this reverts was an incorrect attempted fix for Arm 32, which honestly just made things worse because _CFInfo  just became even more unaligned with CFRuntimeBase. Hence the need to revert.

I'm fine removing this part out of the revert though. As you say, it is Darwin, and I'm more concerned about fixing the bad "fix" as it was originally applied for Linux.